package.json

```diff
@@ -34,12 +34,13 @@
   },
   "homepage": "https://github.com/reduct/registry",
   "devDependencies": {
-    "@reduct/build-tools": "^1.4.5"
+    "@reduct/build-tools": "^1.5.2",
+    "@reduct/logger": "^1.1.0"
   },
   "reduct": {
     "sourceFolder": "Src/",
     "distributionFolder": "Dist/",
     "entryFile": "Registry.js",
-    "globalPackageName": "registry"
+    "globalPackageName": "reductRegistry"
```

Comment:
@Inkdpixels Can you clarify this a little bit? :) This would result in `window.reductRegistry` instead of our namespace `window.reduct.registry`.